rosidl_generator_cpp/test/test_interfaces.cpp

```diff
@@ -34,6 +42,9 @@
 #include "rosidl_generator_cpp/msg/unbounded_array_static.hpp"
 #include "rosidl_generator_cpp/msg/unbounded_array_unbounded.hpp"
 
+#define TEST_VECTOR_SIZE 10
+#define TEST_VECTOR_SIZE_2 3
```

Comment:
Yes, the first one is the size of the primitives array and the second one is the size of nested message array.